src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -59,14 +59,20 @@ def gather_area_loads_and_trade_prices(area, load_price_lists):
 def export_cumulative_loads(area):
     # TODO: Figure out whether this export is needed or can be removed
```

Comment:
Can't we save some resources and bandwidth here, too? Or did you find out that is indeed needed?

Reply:
This variable is queried by the UI, but as far as I checked it is not used anywhere. I will ping Cameron, and if he agrees I will open a separate PR that removes it altogether. Thanks for bringing it up.

Reply:
Thanks! Please ping us what came out from the discussion with Cameron. Thanks!